### PR TITLE
[SOAR-6366] Get logs url change

### PR DESCRIPTION
**Pull request by pmara-r7**
## Proposed Changes

Change insight idr plugin to use correct URL for Get a Log and Get All Logs

### Description

Describe the proposed changes:

  - Change url us.api.insight.rapid7.com to have the extension "log_search" (us.api.insight.rapid7.com/log_search)

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [x] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [x] Screenshot of job output with the plugin changes
- [x] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [x] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [x] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [x] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [x] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [x] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [x] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [x] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [x] Work fully completed
- [x] Functional
  - [x] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [x] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [x] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [x] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [x] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

**Comment by pmara-r7** [comment]
Code should be done, working on getting the required test screenshots now

**Comment by pmara-r7** [comment]
See SOAR-6366 in JIRA for pre-fix behavior. The screenshots below demonstrate a test working. 

connectionTest

workflowtest

I did not add unit tests because the new functionality added requires the use of credentials and an active connection.

File: plugins/rapid7_insightidr/komand_rapid7_insightidr/util/endpoints.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ class QueryLogs:
     @staticmethod
     def get_query_logs(console_url: str, log_id: str):
         """
-        URI for adding get_query_logs
+        URI for adding get_query_logsœ
```
Comment (lmilby-r7) [inline plugins/rapid7_insightidr/komand_rapid7_insightidr/util/endpoints.py:69]:
```suggestion
        URI for adding get_query_logs
```

**Review by lmilby-r7 — CHANGES_REQUESTED** [review]
odd character but looks good

File: plugins/rapid7_insightidr/komand_rapid7_insightidr/util/endpoints.py (hunk 1 above)
Comment (pmara-r7) [reply plugins/rapid7_insightidr/komand_rapid7_insightidr/util/endpoints.py:69]: Year I have no idea how that happened, thanks

**Comment by pmara-r7** [comment]
I have output from the unit test I added but I'm hesitant to upload it here, I'm not sure if it has any sensitive info

File: plugins/rapid7_insightidr/unit_test/test_get_all_logs.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            with open("../tests/get_all_logs.json") as file:
+                test_json = json.loads(file.read()).get("body")
+                connection_params = test_json.get("connection")
+                action_params = test_json.get("input")
+        except Exception as e:
+            self.fail("Likely could not find tests in test directory. Generate and fill out samples to fix this.")
+        return action_params, connection_params, test_action, test_conn
+
+    def test_integration_advanced_query_log_set(self):
+        action_params, connection_params, test_action, test_conn = self.setup()
```
Comment (lmilby-r7) [inline plugins/rapid7_insightidr/unit_test/test_get_all_logs.py:31]:
```suggestion
        action_params, connection_params, test_action, test_conn = self.setUp()
```

**Review by lmilby-r7 — CHANGES_REQUESTED** [review]

**Review by aholland-r7 — APPROVED** [review]

**Pull request merged**